cmd/tailscale/cli/web.go

```diff
@@ -83,6 +85,35 @@ var webArgs struct {
 	cgi    bool
 }
 
+func tlsCertFromEnvironment(listen string) (*http.Server, error) {
+	crt := os.Getenv("TLS_CRT")
+	key := os.Getenv("TLS_KEY")
+	if crt != "" && key != "" {
+		fmt.Println(crt)
+		// We support passing in the complete certificate and key from environment
+		// variables because pfSense stores its cert+key in the PHP config. We populate
+		// TLS_CRT and TLS_KEY from PHP code before starting tailscale web.
+		cert, err := tls.X509KeyPair([]byte(crt), []byte(key))
+		if err != nil {
+			return nil, err
+		}
+
+		tlsConfig := &tls.Config{
+			Certificates: []tls.Certificate{cert},
+		}
+
+		// Build a server:
```

Comment:
comment doesn't add anything. the server and tlsConfig variables don't much either. could just `return &http.Server{...}, nil` probably

Reply:
Made it return tls.Config.